tests/steps/test_docstring.py

```diff
@@ -0,0 +1,201 @@
+import textwrap
+
+from src.pytest_bdd.utils import collect_dumped_objects
+
+
+def test_steps_with_docstrings(pytester):
+    pytester.makefile(
+        ".feature",
+        docstring=textwrap.dedent(
+            '''
+            Feature: Docstring
+
+              Scenario: Step with plain docstring as multiline step
+                Given a step has a docstring
+                """
+                This is a given docstring
+                """
+
+                When a step provides a docstring with lower indentation
+                """
+            This is a when docstring
+                """
+
+                And this step has no docstring
+
+                Then this step has a greater indentation
+                """
+                        This is a then docstring
+                """
+            '''
+        ),
+    )
+
+    pytester.makeconftest(
+        textwrap.dedent(
+            r"""
+        from pytest_bdd import given, when, then
+        from pytest_bdd.utils import dump_obj
+
+
+        @given("a step has a docstring")
+        def _(docstring):
+            given_docstring = docstring.strip()
```

Comment:
why do we call `.strip()`? it shouldn't be necessary right?

Reply:
Absolutely right!  I just got strip()-happy!  Will remove